build.sbt

```diff
@@ -71,11 +71,15 @@ ThisBuild / credentials += Credentials("GnuPG Key ID", "gpg", "F68F0EADDB81EF533
 
 ThisBuild / scalacOptions ++= {
   CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, 13)) => Nil
+    case Some((2, 13)) => Seq(
+      "-deprecation"
+    , "-Wconf:cat=deprecation&msg=Auto-application:silent"
```

Comment:
@shmishleniy Do you have any ideas why we have errors like these?
```
Auto-application to `()` is deprecated. Supply the empty argument list `()` explicitly to invoke method result
[warn] or remove the empty argument list from its definition (Java-defined methods are exempt).
[warn] In Scala 3, an unapplied method like this will be eta-expanded into a function.
[warn]       implicit val ac = caseCodecAuto[ClassWithMap]
```